### DISCUSSION TITLE
Want Sumatrapdf to support the OFD format documents.

### DISCUSSION THREAD

**Discussion by zh-y-h** [sumatrapdfreader/sumatrapdf#4395] [Ideas]
The OFD （i.e. Open Fixed-Layout Document)) is a PDF-liked format. It was developed for electric invoice, government documents, etc, by China.  The size of document under the format is smaller……

Thanks.

**Comment by kjk** [comment]
This is unlikely to happen unless there is an open-source implementation of OFD format I could use.

**Reply by zh-y-h** [reply]
The below may give some helps.
https://github.com/ofdrw/ofdrw

**Comment by GitHubRulesOK** [comment]
> The OFD （i.e. Open Fixed-Layout Document)) is a PDF-liked format. It was developed for electric invoice, government documents, etc, by China. The size of document under the format is smaller……
> 
> Thanks.

It nothing whatsoever like a PDF (nowhere near its format) it is like a <strike>Russian Fiction Book ZIP e.g (FBZ)</strike> it is a zip of XML text files but then again nothing like the Russian format either. Actually its nearest to an ODT and that's nearer to a DocX

![image](https://github.com/user-attachments/assets/3be01155-a159-410f-ba2e-349f8344252f) 
![image](https://github.com/user-attachments/assets/10ecb9ff-b67f-4596-aca5-f995eaea9f71)

![image](https://github.com/user-attachments/assets/e47c8b93-8d7d-433a-9b84-09f81fe7fb86)
![image](https://github.com/user-attachments/assets/e4d85a2a-7811-4cea-86aa-c30bfe4fbd9a)

Here is an ODF Office Document Format for comparison
![image](https://github.com/user-attachments/assets/0b8897d4-9095-4364-b1a9-0c52e80f63f6)

**Comment by ramiropistoia** [comment]
The OXPS/XPS format would also be good, since Microsoft has removed the reader.

**Reply by kjk** [reply]
we do support XPS

**Reply by GitHubRulesOK** [reply]
Even XPS online
![image](https://github.com/user-attachments/assets/ee15eb41-e0dd-43ec-980d-0106ee78a37f)
